src/middleware/requireUser.ts

```diff
@@ -0,0 +1,21 @@
+import { Response, NextFunction } from 'express';
+import { UnauthorizedError } from '../errors';
+import type { AuthenticatedRequest } from '../types';
+
+const requireUser = (
```

Comment:
I'm not quite sure if there's a great reason to differentiate between `requireAuthentication` and `requireUser`

If a user isn't populated then I don't think the authentication was actually properly created.  Given that I'm thinking it would be better to just update `requireAuthentication` to check for the user status.

Reply:
(I took this out)